ads/google/a4a/utils.js

```diff
@@ -846,6 +846,9 @@ export function getBinaryTypeNumericalCode(type) {
       'control': '1',
       'canary': '2',
       'rc': '3',
+      'experimentA': '10',
+      'experimentB': '11',
+      'experimentC': '12',
```

Comment:
Should be include `mod` and `nomod` while we're at it?

Reply:
yeah, lets go ahead and do this.

Reply:
i'll take up the 4x range (i've been using the RC build so its 43)

Reply:
Happy to add that. @erwinmombay to confirm is 
```
mod: 43
nomod: 44
```
?

Reply:
so they are all essentially 43 LOL. they all have different versions, because unlike the rtv experiments they are deployed individually. the way i distinguish them differently is their custom version mark which is the last digit in the rtv (usually defaults to 0).

i can make changes to the deployment if we really need different prefixes between esm/sp/mp

Reply:
LOL,  I wonder how 43 is selected? Why not 42 ?😛 

I don't think we need different prefixes. Also the number here doesn't need to match 43. It's just an id to understand which version of runtime is used to served the A4A. As long as we agree on the id there, it should be fine. Especially that unlike exp_a/b/c, the mod/nomod experiment is temporary.

What about `nomod: 42`, and `mod: 43` to match the style `control (even): exp (odd)`.  Let me know if every agrees on the id. Thanks

Reply:
yeah im ok with that. its 43 because i wanted to take the 4x range, and 3 cause rc is original 03 🤣

Reply:
🤣🤣🤣 
Added `mod` and `nomod` type. PTAL